Samples/All/BumbleRumble/Common/GameStateManager.cpp

```diff
@@ -18,6 +18,24 @@
 
 using namespace BumbleRumble;
 
+namespace
+{
+    std::string NewGuid()

```

Comment:
we seem to drag useful utility code around like this.  could this be put in a shared utility somewhere?  like StringUtils or something?